src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js

```diff
@@ -37,9 +37,7 @@ const ActivityThreadReplies = ({
     const lastReply = replies[replies.length - 1];
 
     const getReplyPermissions = (reply: CommentType): BoxCommentPermission => {
-        const {
-            permissions: { can_delete = false, can_edit = false, can_resolve = false },
-        } = reply;
+        const { permissions: { can_delete = false, can_edit = false, can_resolve = false } = {} } = reply;
```

Comment:
In what scenario would `permissions` be undefined? Its type indicates it's a required field. If this is for testing purposes only, we should avoid making logical changes to support tests.

Reply:
When submitting reply, temporarily new comment is created, and displayed before the response with actual comment get back. This temporal comment doesn't have `permissions` field.
I think alternatively we can fix it by adding `permissions` here
https://github.com/box/box-ui-elements/blob/6b59624dcd33f95eb24193fce1683948f8fcf6c1/src/api/Feed.js#L1707-L1731